### ISSUE TITLE
What is this mess of a software?

### ISSUE THREAD

**Issue by PPPDUD** [bigspeedfpv/blazer#1]
I feel like 90% of this repo is just useless junk to confuse the user. Just give us the code and the prepared builds, not other stuff.

Also: Sir, your takedown notice has arrived. Stop or else you will be blacklisted from my  
softwares.

**Comment by themysticsavages** [comment]
No offense, but please stop embarrassing yourself. You use PyInstaller which basically just packs Python files with a runtime, which is slower and larger to distribute than a precompiled Go app. 

> Also: Sir, your takedown notice has arrived. Stop or else you will be blacklisted from my
softwares.

You don't present yourself as a legal figure in any way, and false DMCAs can be countered. You are known for like three softwares, so blacklisting `blazer` doesn't make a difference.

**Comment by PPPDUD** [comment]
And what are these softwares exactly?

**Comment by PPPDUD** [comment]
Btw, I mean programs that require login will blacklist your username.

**Comment by themysticsavages** [comment]
> Btw, I mean programs that require login will blacklist your username.

OK, but these programs would have to be so useful that it would reach the attention of bigspeed right?

**Comment by PPPDUD** [comment]
Who doesn't want to become part of a history of high-follower users hosted
by me, right?

**Comment by themysticsavages** [comment]
> Who doesn't want to become part of a history of high-follower users hosted by me, right?

Not me, probably not bigspeed either :trollface: Sorry

**Comment by PPPDUD** [comment]
Well, sorry that you don't get the title of more followers than me on my
project.

**Comment by bredcat999** [comment]
> I feel like 90% of this repo is just useless junk to confuse the user. Just give us the code and the prepared builds, not other stuff.
> 
> Also: Sir, your takedown notice has arrived. Stop or else you will be blacklisted from my softwares.

Me when I will have any number of legal things done to me at any given time (it’s very scary and legal)

**Comment by lopste** [comment]
> I feel like 90% of this repo is just useless junk to confuse the user. Just give us the code and the prepared builds, not other stuff.

my honest reaction when I work in an actual programming environment (I do not know how packaging environments work, i just want to go back to my gosh darn single python file :(( )

**Comment by themysticsavages** [comment]
> Well, sorry that you don't get the title of more followers than me on my project.

TLDR: Make actually useful software before saying that a Go program is useless junk

If you are dedicated enough, you could make software that is original and doesn't require lots of packages to do everything for you!

**Comment by PPPDUD** [comment]
Like what? I've already written a program and published it which crawls
Scratch users and lists them.

**Comment by themysticsavages** [comment]
> Like what? I've already written a program and published it which crawls Scratch users and lists them.

Sorry to be blunt, but that's like a beginner program you'd find in a [dummies book](https://www.dummies.com/). Not only does it have minimal utility toward a small audience, but it also depends on scratchclient. You should make a program with a useful and unique intent and I mean so in a motivational way. (please don't take this the wrong way)

**Comment by PPPDUD** [comment]
It doesn't use Scratchclient, it uses ScratchAttach, and also, I can't
believe someone is teaching crawling to a beginner.

**Comment by themysticsavages** [comment]
> I can't believe someone is teaching crawling to a beginner.

Web scraping/crawling is legalized in many countries (including US) and it is basically as simple as requesting data from a server. This is easy to teach to beginners and is the core concept of many products: Google, WaybackMachine, and anything that fetches data from the internet. I mean, `requests` is one of the [most downloaded](https://pypistats.org/top) Python packages, this stuff is commonplace

**Comment by themysticsavages** [comment]
Anyway the point is, you can't take down products with similar intent. It's completely unnecessary on a software with such simple functionality, and this could've all been avoided if you just open-sourced your code. Period.

**Comment by bigspeedfpv** [comment]
> > I feel like 90% of this repo is just useless junk to confuse the user. Just give us the code and the prepared builds, not other stuff.
> 
> my honest reaction when I work in an actual programming environment (I do not know how packaging environments work, i just want to go back to my gosh darn single python file :(( )

mrw i see a properly structured project (what is an `internal`)

can yall not have wars on my profile plz :3

**Comment by Zxnii** [comment]
i fully support you in this take down. slay queen!

**Comment by PPPDUD** [comment]
> i fully support you in this take down. slay queen!

+1
